spec/beaker/subcommand_spec.rb

```diff
@@ -32,6 +32,72 @@ module Beaker
       end
     end
 
+    context 'ensure that beaker options can be passed through' do
+
+      BEAKER_OPTIONS_LIST = [
+        'options-file',
+        'helper',
+        'load-path',
+        'tests',
+        'pre-suite',
+        'post-suite',
+        'pre-cleanup',
+        'provision',
+        'preserve-hosts',
+        'root-keys',
+        'keyfile',
+        'timeout',
+        'install',
+        'modules',
+        'quiet',
+        'color',
+        'color-host-output',
+        'log-level',
+        'log-prefix',
+        'dry-run',
+        'fail-mode',
+        'ntp',
+        'repo-proxy',
+        'add-el-extras',
+        'package-proxy',
+        'validate',
+        'collect-perf-data',
+        'parse-only',
+        'tag',
+        'exclude-tags',
+        'xml-time-order',
+        'debug-errors',
+        'exec_manual_tests',
+        'test-tag-exclude',
+        'test-tag-and',
+        'test-tag-or',
+        'xml',
+        'type',
+        'debug',
+      ]
+
+      let( :yaml_store_mock ) { double('yaml_store_mock') }
```

Comment:
style nitpick: remove spaces around parents `let(:yaml_store_mock)`

Reply:
I want to be consistent with the rest of this file, so... 🤷‍♂️

Reply:
ah didn't notice the rest of the file was doing that...oh well